src/main/java/org/gridsuite/sensitivityanalysis/server/service/SensitivityAnalysisInputBuilderService.java

```diff
@@ -113,6 +114,18 @@ private List<IdentifiableAttributes> goGetIdentifiables(EquipmentsContainer filt
         }
     }
 
+    private Integer getComputationCount(Map<String, List<UUID>> ids, UUID networkUuid, Integer containersAttributesCount) {
+        Map<String, List<Integer>> map = filterService.getIdentifiablesCount(ids, networkUuid, null);
```

Comment:
for contingency list ids, you should request the actions-server and not the filter-server, to get all the contingencies ...

Reply:
done